File: UnicoVehicle/UnicoVehicle.DTO/Customer.cs
using System;
namespace UnicoVehicle.DTO
{
    public class Customer
    {
        public int CustomerId { get; set; }
        public User User { get; set; }
        public District District { get; set; }
        public string CustomerName { get; set; }
        public string Address { get; set; }
        public string photograph { get; set; }
        public string DrivingLicenseNumber { get; set; }
        public string StandardIDNumber { get; set; }
        public string IncomeTaxIDNumber { get; set; }
        public string LastITReturn { get; set; }
        public string BankPassbookPhoto { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/Customer.dart
import './Miscellaneous/User.dart';
import './Miscellaneous/District.dart';

class Customer {
  final int? customerId;
  final User? user;
  final District? district;
  final String? address;
  final String? photograph;
  final String? drivingLicenseNumber;
  final String? standardIDNumber;
  final String? incomeTaxIDNumber;
  final String? lastITReturn;
  final String? bankPassbookPhoto;

  Customer({
    this.address,
    this.bankPassbookPhoto,
    this.customerId,
    this.district,
    this.drivingLicenseNumber,
    this.incomeTaxIDNumber,
    this.lastITReturn,
    this.photograph,
    this.standardIDNumber,
    this.user,
  });
}
